REFERENCE.md

```
@@ -6027,6 +6027,73 @@ URL authUrl = client.authentication().getAuthorizationUrl(
 </dl>
 
 
+</dd>
+</dl>
+</details>
+
+<details><summary><code>client.authentication().<a href="https://github.com/scalekit-inc/scalekit-sdk-java/blob/main/src/main/java/com/scalekit/api/AuthClient.java">getLogoutUrl</a>(options) -> URL</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)